internal/pkg/warewulfconf/defaults.go

```diff
@@ -0,0 +1,47 @@
+package warewulfconf
+
+const (
+	defaultDataStore string = "/srv/warewulf"
```

Comment:
@jcsiadal I think the problem may be here.

Reply:
Agreed. I'll add the correct hook to the makefile.